code/modules/instruments/songs/editor.dm

```diff
@@ -0,0 +1,252 @@
+/**
+  * Returns the HTML for the status UI for this song datum.
+  */
+/datum/song/proc/instrument_status_ui()
```

Comment:
Should have made it tgui with local storage sound db if you're changing it all anyway.

Reply:
tgui scary, if you want it to be a requirement for merge i guess i can screw around and see if i can implement this maybe although i'm not good enough at javascript to do local storage yet unless there's some API in tgui 4.0 i'm not aware of